packages/cli/src/__tests__/commands/Init.test.ts

```diff
@@ -25,137 +22,119 @@ test('works with url param', async () => {
   expect(stripAnsi(result.stdout)).toMatchSnapshot()
 
-  const schema = fs.readFileSync(
-    join(ctx.tmpDir, 'prisma', 'schema.prisma'),
-    'utf-8',
-  )
+  const schema = fs.readFileSync(join(ctx.tmpDir, 'prisma', 'schema.prisma'), 'utf-8')
   expect(schema).toMatch(defaultSchema('postgresql'))
 
   const env = fs.readFileSync(join(ctx.tmpDir, '.env'), 'utf-8')
   expect(env).toMatchInlineSnapshot(`
-# Environment variables declared in this file are automatically made available to Prisma.
-# See the documentation for more detail: https://pris.ly/d/prisma-schema#using-environment-variables
+    # Environment variables declared in this file are automatically made available to Prisma.
+    # See the documentation for more detail: https://pris.ly/d/prisma-schema#using-environment-variables
 
-# Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server and MongoDB (Preview).
-# See the documentation for all the connection string options: https://pris.ly/d/connection-strings
+    # Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server and MongoDB (Preview).
+    # See the documentation for all the connection string options: https://pris.ly/d/connection-strings
 
-DATABASE_URL="postgresql://johndoe:randompassword@localhost:5432/mydb?schema=public"
-`)
+    DATABASE_URL="postgresql://johndoe:randompassword@localhost:5432/mydb?schema=public"
+  `)
 })
 
 test('works with provider param - mysql', async () => {
   ctx.fixture('init')
   const result = await ctx.cli('init', '--datasource-provider', 'mysql')
   expect(stripAnsi(result.stdout)).toMatchSnapshot()
 
-  const schema = fs.readFileSync(
-    join(ctx.tmpDir, 'prisma', 'schema.prisma'),
-    'utf-8',
-  )
+  const schema = fs.readFileSync(join(ctx.tmpDir, 'prisma', 'schema.prisma'), 'utf-8')
   expect(schema).toMatch(defaultSchema('mysql'))
 
   const env = fs.readFileSync(join(ctx.tmpDir, '.env'), 'utf-8')
   expect(env).toMatchInlineSnapshot(`
-# Environment variables declared in this file are automatically made available to Prisma.
-# See the documentation for more detail: https://pris.ly/d/prisma-schema#using-environment-variables
+    # Environment variables declared in this file are automatically made available to Prisma.
+    # See the documentation for more detail: https://pris.ly/d/prisma-schema#using-environment-variables
 
-# Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server and MongoDB (Preview).
-# See the documentation for all the connection string options: https://pris.ly/d/connection-strings
+    # Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server and MongoDB (Preview).
+    # See the documentation for all the connection string options: https://pris.ly/d/connection-strings
 
-DATABASE_URL="mysql://johndoe:randompassword@localhost:3306/mydb"
-`)
+    DATABASE_URL="mysql://johndoe:randompassword@localhost:5432/mydb"
```

Comment:
port looks wrong - side effect from new snapshot logic?

Reply:
Nice find, I'll try a fix in the logic